src/as/core/Clock.spec.wat

```diff
@@ -0,0 +1,5083 @@
+(module
```

Comment:
Did you mean to commit this one?

Reply:
I recommend inspecting the wat files. They helped me debug a lot of problems.

Reply:
Are they generated files?